app/models/post.rb

```diff
@@ -0,0 +1,15 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments, foreign_key: :author_id
+  has_many :likes, foreign_key: :author_id
+
+  after_save :update_user_posts_counter
+
+  def update_user_posts_counter
+    author.increment!(:posts_counter)
+  end
+
```

Comment:
- [ ] Could you make this method private since it is only called here, as requested by the previous reviewer? :pray: